tests/README.md

```diff
@@ -1,5 +1,79 @@
-# Python Tests against the arkouda_server
+# Python Tests against the arkouda\_server
 
-  * depending on your setup you may need to set `PYTHONPATH` environment variable to find the `arkouda.py` module
-  * check the python source for runtime options
-  
\ No newline at end of file
+All Python tests can be run against a local or remote arkouda\_server. The arkouda project is configured so that
+no special Python paths need to be defined.
+  
+# The arkouda Python test harness
+
+The arkouda project has a [pytest](https://docs.pytest.org/en/latest/)-based Python test harness that can be executed 
+via the command-line interface (cli) or within the make file. 
+
+## pytest environment preparation
+
+There are two python libraries that must be installed to execute the arkouda test harness, pytest and pytest-env,
+both of which can be installed via pip:
+```
+pip install pytest, pytest-env
```

Comment:
Add these to arkouda test dependencies ? 

```diff
diff --git a/setup.py b/setup.py
index 7a814a0..313ff93 100644
--- a/setup.py
+++ b/setup.py
@@ -149,7 +149,7 @@ setup(
     extras_require={  # Optional
         'doc': ['Sphinx', 'sphinx-argparse'],
         'dev': ['h5py'],
-        'test': ['pandas', 'pexpect'],
+        'test': ['pandas', 'pexpect', 'pytest', 'pytest-env'],
     },
     # replace orginal install command with version that also builds
     # chapel and the arkouda server.
```

Reply:
sure, good point!